src/lib/components/organisms/scanner/Scanner.svelte

```diff
@@ -57,9 +57,9 @@
 	<ion-toolbar>
 		<div class="flex flex-row">
 			<ion-title>{m.QR_SCAN()}</ion-title>
-			<button on:click={stopScan} on:keydown={stopScan} aria-hidden>
+			<ion-button on:click={stopScan} on:keydown={stopScan} aria-hidden fill="clear">
 				<ion-icon icon={close}></ion-icon>
-			</button>
+			</ion-button>
```

Comment:
Why did we go back? Is something missing from the implementation on the components? Wouldn't it be better to implement it there instead?